datasets/lis.da.trend.data.mdx

```diff
@@ -11,7 +11,15 @@ media:
 taxonomy:
   - name: Topics
     values:
```

Comment:
Who suggested these topics? They're all over the map in specificity.

Reply:
I believe EIS has a list they required we include.